subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/configurationcache/ConfigurationCacheProblemsFixture.groovy

```diff
@@ -341,14 +336,12 @@ final class ConfigurationCacheProblemsFixture {
     }
 
     private static void assertProblemsHtmlReport(
-        File rootDir,
```

Comment:
💅 I'd introduce an overload instead (in order to avoid changes to existing call-sites)

Reply:
Reverted that change